plugins/system/userlogs/userlogs.xml

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extension version="3.6" type="plugin" group="system" method="upgrade">
-    <name>PLG_SYSTEM_USERLOGS</name>
-    <author>Joomla! Project</author>
-    <creationDate>May 2016</creationDate>
-    <copyright>Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
+	<name>PLG_SYSTEM_USERLOGS</name>
+	<author>Joomla! Project</author>
+	<creationDate>May 2016</creationDate>
+	<copyright>Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
 	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
 	<authorEmail>admin@joomla.org</authorEmail>
 	<authorUrl>www.joomla.org</authorUrl>
 	<version>0.6.0</version>
```

Comment:
Tobias already changed it in his PRs, as you don't now when your project gets merged we use now a placeholder (DEPLOY_VERSION) instead of the version at core.